examples/trusted_database/client/rust/src/main.rs

```diff
@@ -72,8 +75,13 @@ async fn main() -> anyhow::Result<()> {
         .await
         .context("Couldn't create TLS channel")?;
     let label = Label::public_untrusted();
-    let interceptor =
+    let label_interceptor =
         LabelInterceptor::create(&label).context("Couldn't create gRPC interceptor")?;
+
+    let key_pair = oak_sign::KeyPair::generate()?;
+    let auth_interceptor = AuthInterceptor::create(key_pair);
+
+    let interceptor = combine(label_interceptor, auth_interceptor);
```

Comment:
Could you please add tests for the new logic into abitest? Maybe in the next commit